views/room.ejs

```diff
@@ -312,6 +312,7 @@
               <span>copied to clipboard!</span>
             </div>
           </div>
+          <%   if(hasSip) { %>
```

Comment:
👍  yup this needed doing. We also need do the same thing for google text when google auth isn't required.